packages/server-wallet/src/models/channel.ts

```diff
@@ -124,6 +127,39 @@ export class Channel extends Model implements RequiredColumns {
       .first();
   }
 
+  static async setLedger(
+    channelId: Bytes32,
+    assetHolderAddress: Address,
+    txOrKnex: TransactionOrKnex
+  ): Promise<void> {
+    await Channel.query(txOrKnex)
+      .findOne({channelId})
+      .patch({assetHolderAddress});
+  }
```

Comment:
I'd be in favour of changing the interface so that channels can't become ledgers after creation: you're either always a ledger, or never a ledger. It seems that by providing the ability for channels to change type, we enable a lot of edge-cases, which it would be easier not to have to think about or handle.